pkg/daemon/ceph/client/image.go

```diff
@@ -63,6 +64,30 @@ func ListImages(context *clusterd.Context, clusterName, poolName string) ([]Ceph
 	return images, nil
 }
 
+func getImageInfo(context *clusterd.Context, clusterName, name, poolName string) (*CephBlockImage, error) {
+	imageSpec := getImageSpec(name, poolName)
+	args := []string{"info", imageSpec}
+	buf, err := ExecuteRBDCommand(context, clusterName, args)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get image %s info: %+v", imageSpec, err)
+	}
+
+	jsonRegex := regexp.MustCompile(`(?m){.+}`)
```

Comment:
Can we safely assume there's no `{...}` in librados debug output?

Reply:
have you rebased on the latest? #2247 should have fixed this problem with debug output so that the ceph daemons would log to stderr instead of stdout. If we have to parse the output like this i'd be worried.

Reply:
I've just tested with `log file = /dev/stderr` in `ceph.conf` and it seems to be working fine; I'll revert the commit. We should probably remove other debugging output related handling like [this one](https://github.com/rook/rook/blob/master/pkg/daemon/ceph/client/image.go#L48-L56).